<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шумков Д.В. 4134к</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>791</width>
      <height>441</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>325</y>
        <width>701</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Письма</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Получатели</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Отправители</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>360</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/src/free-icon-mail-646094.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>360</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/src/free-icon-recipient-9101806.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>360</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/src/free-icon-mailing-10070244.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>581</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/src/photo_2023-05-30_22-47-25.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>380</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавление</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>31</y>
         <width>161</width>
         <height>241</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Адрес</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Телефон</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_7"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>151</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Редактирование</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_8"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>Редактировать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>361</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вы в окне &quot;Получатель&quot;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>400</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>371</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>731</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Отправитель</string>
         </property>
         <widget class="QComboBox" name="comboBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>341</width>
            <height>24</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Получатель</string>
         </property>
         <widget class="QComboBox" name="comboBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>341</width>
            <height>24</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_Send">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Написать письмо</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вы в окне &quot;Письма&quot;</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>401</width>
        <height>216</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Поле ввода сообщения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_Message"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>401</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Тема письма</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Subject"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>390</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>381</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>361</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Добавление</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>151</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Имя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Адрес</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Телефон</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Редактирование</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>151</width>
            <height>141</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>Редактировать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вы в окне &quot;Отправитель&quot;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="dateTimeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </widget>
   <widget class="QCheckBox" name="safeModeCheckBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Безопасный режим</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Мега ультра почта</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
